### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by vepadulano**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Review by eguiraud — APPROVED** [review]
LGTM!

**Comment by henryiii** [comment]
@conda-forge-admin, please rerender

**Comment by vepadulano** [comment]
Let me rebase this branch

**Comment by vepadulano** [comment]
Rebased again after recent merge of 6.32.02. Let's wait for the CI then I believe we should merge this.

File: recipe/meta.yaml
Comment (chrisburr) [inline recipe/meta.yaml:4]: There is no need to bump the build number in this case as we don't want to deploy a new build.

Comment (vepadulano) [reply recipe/meta.yaml:4]: Ah fair enough!

**Comment by vepadulano** [comment]
Great MacOS is broken again :)

File: recipe/meta.yaml
Comment (jakirkham) [inline recipe/meta.yaml:43]: Maybe drop the patch, reset the build number, ad skip CI?

That way this is just maintainer changes and CI is not really needed then

**Pull request merged**
